CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contributing Guidelines
+
+Thanks for interest in contribution to `colin`.
+
+The following is a set of guidelines for contributing to `colin`.
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How to contribute code to colin
+
+### Reporting Bugs
+Before creating bug reports, please check a [list of known issues](https://github.com/user-cont/colin/issues) to see
+if the problem has already been reported.
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/user-cont/colin/issues/new).
+Be sure to include a **descriptive title, clear description and a package version** and please include **as many details as possible** to help maintainers reproduce
+the issue and resolve it faster.
+If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as GitHub issues.
+When you are creating an enhancement issue, **use a clear and descriptive title**
+and **provide a clear description of the suggested enhancement**
+in as many details as possible.
+
+# How to write the new check to colin
+All checks are stored in the directory [checks](https://github.com/user-cont/colin/tree/master/colin/checks).
+Think about what kind of check you would like to write and browse for already existing checks:
+ - [best practices](https://github.com/user-cont/colin/tree/master/colin/checks/best_practices)
+ - [containers](https://github.com/user-cont/colin/tree/master/colin/checks/containers)
+ - [dockerfile](https://github.com/user-cont/colin/tree/master/colin/checks/dockerfile)
+ - [images](https://github.com/user-cont/colin/tree/master/colin/checks/images)
+ - [labels](https://github.com/user-cont/colin/tree/master/colin/checks/labels)
+
+Here's a simple template how you can create a new check:
+
+## Label check example
+```python
+from colin.checks.abstract.labels import LabelCheck
+
+class FooBarLabelCheck(LabelCheck):
+
+    def __init__(self):
+        super(self.__class__, self) \
+            .__init__(name="foobar_label",
+                      message="Label 'foobar' has to be specified.",
+                      description="Provide a description for foobar label.",
+                      reference_url="Specify URL to foobar reference",
+                      tags=["foobar", "label", "required"],
```

Comment:
Please, remove the `required` tag. It is not the issue of this pull request, but we have to remove that also from the real checks. (So, less work with that later.)

The reason is that it does not make sense to set the severity here because it can differ between rulesets.